gcp/tests/test_connect_scheduling.py

```
@@ -0,0 +1,562 @@
+        )
+        assert result.hour == 13
+        assert result.tzinfo is not None
+
+    def test_central_to_utc(self):
+        result = compute_send_at_utc(
+            date(2026, 6, 10), time(9, 0), "US/Central"
+        )
+        assert result.hour == 14
+
+    def test_unknown_timezone_raises(self):
+        with pytest.raises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)